web/themes/custom/drevops/assets/sass/brand/_components.scss

```
@@ -0,0 +1,1712 @@
+.component-reveal-d2 { transition-delay: 0.2s; }
+.component-reveal-d3 { transition-delay: 0.3s; }
+.component-reveal-d4 { transition-delay: 0.4s; }
+.component-reveal-d5 { transition-delay: 0.5s; }
+.component-reveal-d6 { transition-delay: 0.6s; }
+
+/*
+ * DrevOps — Animations
+ *
+ * Shared keyframe definitions for hero glow, scroll indicator,
+ * and entrance transitions.
+ */
```

Stylelint (17.12.0)

[error] Expected // for comments instead of /* (scss/comment-no-loud)

(scss/comment-no-loud)